### ISSUE TITLE
Increase contrast in sidebar

### ISSUE THREAD

**Issue by vickychijwani** [Semantic-Org/Semantic-UI-Docs#277]
Hi, I've started using Semantic UI recently. Overall I really like the framework and am excited to work with it!

I have a suggestion for the docs website: increase the text contrast in the left sidebar, it's really low. 12px font, light grey text on dark grey background. (note: "contrast" is a combination of many things: size, color, weight). Personally, I find it quite hard to read. I also don't prefer zooming in because then the sidebars take up too much horizontal space and the body text becomes slightly too large.

### So this is just your opinion?

There's tons of research and guidelines for text contrast on the web, see the [W3C guidelines][w3c] and the cheeky-but-effective [Contrast Rebellion manifesto](http://contrastrebellion.com/). I think that increasing the contrast should make the docs more readable for a significant number of people with low-quality displays/color deficiency (like me)/vision loss/large font preference.

I actually calculated the contrast ratio (as per the [W3C guidelines][w3c]) and it [works out to 5.2:1](http://leaverou.github.io/contrast-ratio/#rgba%28255%2C255%2C255%2C.5%29-on-%231b1c1d). The [minimum color contrast recommended](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html) is 4.5:1, so at first glance it seems we're ok. But here's the catch: the guideline applies to _font sizes >= 24px/18pt_ (at regular weight). Semantic's sidebar has a font size of 12px. So we either _barely_ clear the bar, or not at all. And this is the _minimum_ we're talking about.

I can send a PR with several variations to improve the contrast, but first I need some feedback and consensus on this. Thanks for reading!

[w3c]: https://www.w3.org/TR/WCAG/#visual-audio-contrast

**Comment by IonutBajescu** [comment]
Agreed. While I haven't had any problems with the sidebar's contrast back at home on my macbook, now that you pointed it out it is indeed harder to read on the monitors we use at work.
